pkg/providers/instance/instance.go

```diff
@@ -893,6 +893,7 @@ func (p *DefaultProvider) getCSExtension(cse string, isWindows bool) *armcompute
 				"commandToExecute": cse,
 			},
 		},
+		Tags: tags,
```

Comment:
We should also add tags to the AKSIdentifyingExtension if we don't already.

Reply:
Added the tags to the AKSIdentifyingExtension.